views/default/elements/layout/topbar.css

```diff
@@ -216,6 +216,10 @@
 	order: 3;
 }
 
+.elgg-page-topbar .elgg-menu-site-container .elgg-menu-site .elgg-anchor-icon {
+	display: none;
```

Comment:
This doesn't seem to be enough. I am seeing extra spacing, which pushes the menu out.

Reply:
yep... caused by `.elgg-menu .elgg-anchor-icon+.elgg-anchor-label` adding the space... need to clear that too for the site-menu